Frameworks/CoreText/DWriteWrapper_CoreText.mm

```diff
@@ -52,10 +46,26 @@ static DWRITE_TEXT_ALIGNMENT __CoreTextAlignmentToDwrite(CTTextAlignment alignme
     }
 }
 
+static DWRITE_WORD_WRAPPING __CTLineBreakModeToDWrite(CTLineBreakMode lineBreakMode) {
```

Comment:
hmmm. Seems like we should be more confident with this. How do the unit tests build? Are these items in a .def file? Would a private dll export have any other potential pitfalls with regards to encapsulation and versioning? 

---
In reply to: [90936774](https://github.com/Microsoft/WinObjC/pull/1504#discussion_r90936774) [](ancestors = 90936774,90918451,90746929,90742638)